/home/alex/code/machineconfig/src/machineconfig/jobs/python_custom_installers/dev/espanso.py
  /home/alex/code/machineconfig/src/machineconfig/jobs/python_custom_installers/dev/espanso.py:71:27 - error: Cannot access attribute "from_dict" for class "type[Installer]"
    Attribute "from_dict" is unknown (reportAttributeAccessIssue)
1 error, 0 warnings, 0 informations
